cumulusci/tasks/preflight/sobjects.py

```diff
@@ -0,0 +1,101 @@
+from cumulusci.tasks.salesforce import BaseSalesforceApiTask
+from cumulusci.core.exceptions import TaskOptionsError
+from cumulusci.core.utils import process_list_arg
+
+from simple_salesforce.exceptions import SalesforceMalformedRequest
+
+
+class CheckSObjectsAvailable(BaseSalesforceApiTask):
+    api_version = "48.0"
+
+    task_options = {
+        "sobjects": {
+            "description": "A list of sObjects whose presence needs to be verified.",
+            "required": True,
+        }
+    }
+
+    def _run_task(self):
+        all_objects = {entry["name"] for entry in self.sf.describe()["sobjects"]}
+        self.return_values = all(
+            sobject in all_objects
+            for sobject in process_list_arg(self.options["sobjects"])
+        )
```

Comment:
I'd suggest setting return_values to a list of all the sobjects. That way the return value can be cached and used for multiple preflight checks (e.g. `when: "Network" not in tasks.check_sobjects_available()`). If multiple checks need to call the task with different options, it can't be cached.

Reply:
Done!